api-gateway-routes/api_gateway_proxy_function.rb

```diff
@@ -80,6 +80,13 @@ def on_connect(event, context)
   elsif authorizer['mini_app_type'] == 'theater'
     function_name = ENV['BUILD_AND_RUN_THEATER_PROJECT_LAMBDA_ARN']
   else
+    # log so we know we saw an invalid mini app
+    log_data = {
+      connectionId: request_context["connectionId"], 
```

Comment:
Maybe worth pulling this out into its own function and using it in other spots where we log (lines 40 and 45) so the log format is consistent?

Reply:
good idea

Reply:
updated all the `puts` in this function to use the new formatting